lib/fast_jsonapi/multi_to_json.rb

```diff
@@ -43,9 +43,20 @@ def rescue
       end
     end
 
-    def self.logger(device=nil)
-      return @logger = Logger.new(device) if device
-      @logger ||= Logger.new(IO::NULL)
+    def self.to_json(object)
+      _fast_to_json(object)
+    rescue NameError
+      define_to_json(FastJsonapi::MultiToJson)
+      _fast_to_json(object)
+    end
+
+    private
```

Comment:
I think what you might mean here is to apply `# @api private` to the below methods, which I disagree with. It might make more sense to document intended usage, but I would say this is all a stable public API.

The private keyword has no effect on the interface.

In any case, the argument I can see for declaring it `# @api private` would be to say to anyone using the library not to rely on those methods being there, whether or not their public.  (In Ruby, nothing's really private anyhow).